sphinx/build_docs.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+sphinx-build -b html "sphinx/source" "docs"
+touch "/docs/.nojekyll"
```

Comment:
Without knowing how this script is used by Sphinx, I'd prefer to place this script in the bin folder. Do you know what is the exact goal of this script here and how it should be used when documenting the code?

Reply:
Yes, this script generates the docs folder containing the html documentation. Anyway, I can move it to bin

Reply:
won't you be using "make html" for doing this?